pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.29</version>
+        <version>4.4</version>
```

Comment:
```suggestion
        <version>2.29</version>
```

Reply:
This is required for `aws-credentials` plugin and `aws-java-sdk`